README.md

```diff
@@ -30,15 +32,45 @@ to ensure that compliance is granted only to the exact binary tested.
 FIPS compliance was [granted](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4407)
 to an old version of BoringSSL that was tested with `Clang 12.0.0`.
 
-Installing `Clang 12.0.0` on modern environments is at best difficult. For `linux` `x86_64`, we work around
-this problem by shipping the pre-built binaries under `vendor/boringssl-fips/linux_x86_64`. We then tell the BoringSSL libraries
-to use this path by setting the `BORING_BSSL_PATH` environment variable in `.cargo/config.toml`.
+Given that FIPS support will always have special environmental build requirements, we currently we work around this by vendoring OS/arch specific FIPS-compliant binary builds of `boringssl` in [](vendor/boringssl-fips/)
+
+We vendor FIPS boringssl binaries for
+- `linux/x86_64`
+- `linux/arm64`
+
+To use these vendored libraries and build ztunnel for either of these OS/arch combos, for the moment you must manually edit
+[.cargo/config.toml](.cargo/config.toml) and change the values of BORING_BSSL_PATH and BORING_BSSL_INCLUDE_PATH under the `[env]` key to match the path to the vendored libraries for your platform, e.g:
+
+#### For linux/x86_64
+
+``` toml
+BORING_BSSL_PATH = { value = "vendor/boringssl-fips/linux_x86_64", force = true, relative = true }
+BORING_BSSL_INCLUDE_PATH = { value = "vendor/boringssl-fips/linux_x86_64/include/", force = true, relative = true }
+```
+
+#### For linux/arm64
+
+``` toml
+BORING_BSSL_PATH = { value = "vendor/boringssl-fips/linux_arm64", force = true, relative = true }
+BORING_BSSL_INCLUDE_PATH = { value = "vendor/boringssl-fips/linux_arm64/include/", force = true, relative = true }
```

Comment:
nit: is the include/ arch specific or can we share it?

Reply:
Yeah I thought about that, I think so, lemme give that a go.

Reply:
I put a hold in case you want to do this but approved, feel free to remove

Reply:
Yep - consolidated, ty for asking.